src/client/components/Modal/Modal.stories.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
+import Modal from './Modal.component';
+
+export default { title: 'Components / Modal', component: Modal };
+function testFunction() {
+  console.log('test function');
+}
+const Template = (args) => {
```

Comment:
Also this fails the quick fix for this is to change into : `function Template(args) {...}`